app/records-interfaces/bucket-records-interface.coffee

```diff
@@ -0,0 +1,13 @@
+null 
+
+### @ngInject ###
+global.cobudgetApp.factory 'BucketRecordsInterface', (BaseRecordsInterface, BucketModel) -> 
+  class BucketRecordsInterface extends BaseRecordsInterface
+    model: BucketModel
+    constructor: (recordStore) ->
+      @baseConstructor recordStore
+      @restfulClient.apiPrefix = 'http://localhost:3000/api/v1'
```

Comment:
i also recommend against versioning the api in this way, but that's more an issue for the api.